src/targets/gpu/CMakeLists.txt

```diff
@@ -22,43 +22,135 @@
 # THE SOFTWARE.
 # ####################################################################################
 
-list(APPEND CMAKE_PREFIX_PATH /opt/rocm)
```

Comment:
This shouldn't be removed.

Reply:
The `CMAKE_PREFIX_PATH` is set in upper CMake already. It is redundant information.